pkg/security/token.go

```diff
@@ -37,6 +37,11 @@ func GetAppToken() string {
 	return os.Getenv(consts.AppAPITokenEnvVar)
 }
 
+// GetAppToken returns the value of the app x-api-key from an environment variable.
+func GetAppKey() string {
+	return os.Getenv(consts.APIKeyHeader)
+}
```

Comment:
Thank you @JoshVanL  for the review . doubt: In both cases we would like to set it in empty string right ? since if we do pass something as null there is a potential that we might error out somewhere

Reply:
Hm do we know that for sure?

I'm quite keen to have the functionality that if the env var is _set_, we do something with the value.

Reply:
Sorry I dint frame correctly , I meant in both cases whether its ```os.Lookup``` or ```os.Getenv``` if value is not there we would just leave it empty right ? because in main code we take action by checking if its not empty .

Reply:
Indeed, I'm suggesting that we set or don't set the header based on the environment variable being defined, not on whether its value is empty or not.

Reply:
The reason I say this is because I believe it to be more proper to respect an environment variable being set, rather than being "smart" as to what the value is. We can return `(string, bool)` here, or ideally, `*string` where the `nil` value signals the environment variable isn't set.

Reply:
In this case, not set or empty is the same behavior. We cannot set the header name as empty string, so we default to the default value, just like when it is not set.